src/Persistence/Models/Control.cs

```diff
@@ -50,15 +45,10 @@ public Control(string name)
     /// <summary>
     /// key/value pairs of Control properties. Mapped to/from Control rules.
     /// </summary>
-    [YamlMember(Order = 2)]
     public ControlPropertiesCollection Properties { get; init; } = new();
 
     /// <summary>
     /// list of child controls nested under this control.
-    /// </summary>
-    [YamlMember(Order = 3)]
-    public IList<Control> Controls { get; init; } = new List<Control>();
-
-    [YamlIgnore]
-    public ControlEditorState? EditorState { get; set; }
```

Comment:
It is marked as YamlIngore. why do you need to delete it?

Reply:
we're not loading Editor state here for the moment